src/main/java/org/spongepowered/api/service/economy/PhysicalCurrency.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.service.economy;
+
+import org.spongepowered.api.item.ItemType;
+
+/**
+ * Represents a {@link Currency} which is backed by inventory items.
+ * 
+ * <p>Physical currencies have a linked {@link ItemType} that represents how the
+ * currency will show up in inventory when it is held.  This does not mean that 
+ * if an account holds the items that they are currently inside of an inventory.</p>
+ *
+ * @see Currency
+ */
+public interface PhysicalCurrency {
+
+    ItemType getItemType();
```

Comment:
The only way I can see it working for multiple currencies is for the economy implementation to abstract it away like they did with Vault. But at least now the transactions and exchanges have detailed failure reasons so it should be workable.

For physical currencies it might be better to just ignore the itemTypes and have methods to convert between the number form of the currency and a set of ItemStacks. It wouldn't need to guarantee a single type.

You could also have methods to take a given inventory and convert it to it's currency counterpart, but you would have to deal with change/rejected items.

That seems the most flexible solution.